src/components/IOUBadge.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, TouchableOpacity} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import Num from 'expensify-common/lib/Num';
+import ONYXKEYS from '../ONYXKEYS';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // IOU Report data object
+    iouReport: {
+        // The total amount in cents
+        total: PropTypes.number,
+    },
+};
+
+const defaultProps = {
+    iouReport: {
+        total: 0,
+    },
+};
+
+const IOUBadge = props => (
+    <TouchableOpacity
+        style={[styles.pill, styles.ml2]}
+    >
```

Comment:
As we've descoped the tap to scroll feature for now I think the TouchableOpacity should also be removed.